bin/compile

```diff
@@ -314,6 +297,35 @@ composer validate --no-check-publish --no-check-all --quiet "$COMPOSER" 2>/dev/n
 	EOF
 }
 
+# if prefer-stable is false and minimum-stability is not stable, warn about potential unstable platform installs
+[[ -f "$COMPOSER_LOCK" ]] && cat "$COMPOSER_LOCK" | python -c 'import sys, json; l = json.load(sys.stdin); sys.exit(l.get("minimum-stability", "stable") != "stable" and l.get("prefer-stable", False) == False);' 2> /dev/null || {
+	mcount "warnings.composer_lock.minimum_stability"
+	warning <<-EOF
+		Non-stable 'minimum-stability'!
```

Comment:
This warning reads better without this first line I think. The next paragraph is a better place to start.

Reply:
The "heading" is intentional; the first line is re-printed in a warnings summary in case an error occurs.

Reply:
Then I'd suggest changing `Non-stable 'minimum-stability'!` to something a little more friendly? 🤔 

`Configuration could lead to using unstable versions of PHP` maybe?

Reply:
It's supposed to be unfriendly, as it could result in people getting really unexpected stuff. This message used to show up for anyone with `minimum-stability` other than `stable`, regardless of `prefer-stable`, which is no longer the case, so the users who now still see it really need to act on it, otherwise they'll get a PHP 7.3 RC once it's out.